test/unit/webgl/p5.Shader.js

```diff
@@ -67,7 +67,8 @@ suite('p5.Shader', function() {
       var expectedAttributes = ['aPosition', 'aNormal', 'aTexCoord'];
 
       var expectedUniforms = [
-        'uModelViewMatrix',
+        'uModelMatrix',
```

Comment:
I think we can keep these the same too -- we still use `uModelViewMatrix` for shaders here: https://github.com/processing/p5.js/blob/b9828a71c63204c08e4e7c6435995f9603ed3044/src/webgl/p5.Shader.js#L339

...and although now we also pass in `uModelMatrix` and `uViewMatrix` for user shaders that need it, these tests only assert that uniforms are present in our existing shaders, which don't use those yet.